pkg/virt-api/webhooks/hyperv.go

```diff
@@ -199,10 +199,10 @@ func ValidateVirtualMachineInstanceHypervFeatureDependencies(field *k8sfield.Pat
 	return causes
 }
 
-func SetVirtualMachineInstanceHypervFeatureDependencies(vmi *v1.VirtualMachineInstance) error {
+func SetVirtualMachineInstanceHypervFeatureDependencies(spec *v1.VirtualMachineInstanceSpec) error {
```

Comment:
Rename to `SetVirtualMachineInstanceSpecHypervFeatureDependencies`?

Reply:
Or get rid of `VirtualMachineInstance` in the name to keep it short?

Reply:
Yeah I've just dropped `VirtualMachineInstance` from most of the names now, thanks!